changelogs/unreleased/6526-index-float-int.yml

```
@@ -0,0 +1,9 @@
+description: number type should no longer differentiate between integer and decimal notation when used in indexes.
+issue-nr: 6526
+issue-repo: inmanta-core
+change-type: minor
+destination-branches: [master]
+sections:
+  feature: "Introduce the 'float' type which is like a float in python"
```

I would not make a comparison with Python here, just say what it is.
```suggestion
  feature: "Introduce the 'float' type for floating point numbers"
```